src/utils/XhrNetworkInterceptor.ts

```diff
@@ -145,14 +146,20 @@ export default {
 
             // @ts-ignore
             if (this._hasError) {
-              cloneNetwork.errorCode = 0;
+              cloneNetwork.errorCode = clientErrorCode;
               cloneNetwork.errorDomain = 'ClientError';
 
               // @ts-ignore
               const _response = this._response;
-              cloneNetwork.requestBody =
-                typeof _response === 'string' ? _response : JSON.stringify(_response);
-              cloneNetwork.responseBody = null;
+              // Detect a more descriptive error message.
+              if (typeof _response === 'string' && _response.length > 0) {
+                cloneNetwork.errorDomain = _response.replaceAll(' ', '-');
```

Comment:
We are still using `-` instead of spaces because of the backend limitation, right? If so, do we have a backlog item for changing this in the future?